src/utils/transaction.ts

```diff
@@ -17,20 +22,62 @@ export type AnyTransaction = Transaction<any, string, string, unknown>;
  **/
 export async function submitTransaction(
 	transaction: AnyTransaction,
-	successMessage?: string
+	successMessage?: string,
+	tryUseMultisig?: Bounty | ChildBounty
 ): Promise<TxFinalizedPayload | undefined> {
+	const proxyAddress = tryUseMultisig?.curator;
+	const multisigAddress = tryUseMultisig?.curatorMultisigAccount;
 	try {
-		const signer = get(polkadotSigner);
+		let signer = get(polkadotSigner);
 		if (!signer) {
 			showErrorDialog('Internal Error, signer is undefined.');
 			return;
 		}
 
-		showLoadingDialog('Submitting transaction');
+		const typedApi = get(dotApi);
+		let calldata = undefined;
+
+		const source = get(activeAccount)?.source;
+
+		// In case it's multisig transaction.
+		if (proxyAddress && multisigAddress && source !== 'mimir') {
```

Comment:
Should we also exclude WalletConnect? Or is it used for "normal" addresses as well? But isn’t it true for Mimir too?

Reply:
WalletConnect can also have normal addresses, meaning if you're connected with WalletConnect, you could initialte a Multisig from the BountyManager if you're connected to subwallet for example and not Multix. While Mimir is meant to handle Multisigs only, no?

Reply:
I think I signed in via Mimir using a normal Alice account